resources/check.rb

```diff
@@ -6,7 +6,7 @@
 attribute :command, :kind_of => String, :required => true
 attribute :aggregate, :kind_of => [String, TrueClass, FalseClass]
 attribute :aggregates, :kind_of => Array
-attribute :interval, :default => 60
+attribute :interval, :kind_of => Integer
```

Comment:
This makes this a breaking change and would require lots of people to potentially update their cookbooks. I think this is how we would do it if we had known that `cron` was gonna be a thing when we made the cookbook and decided on a default for interval. I think we want a default of 60 still with a `:kind_of => [Integer, FalseClass` and would require anyone wishing to use `cron` to set `interval` to `false` this prevents a breaking change.

After reaching the end of the PR backwards compatibility is maintained but is less obvious to the user.